pkg/chain/ethereum/_LEGACY_V1

```diff
@@ -1 +1 @@
-This directory contains legacy code from V1. It should be removed.
+This directory contains legacy code from V1. It should be moved to fit the new directories structure.
```

Comment:
I would clear up this directory and remove everything that is not needed for v2, assuming there is a way to make the compilation pass.